apps/backend/src/websocket/connection_manager.py

```
@@ -0,0 +1,249 @@
+        logger.info(f"New WebSocket connection: {client_id}")
+        
+        # Start heartbeat task if this is the first connection
+        if len(self.connections) == 1 and not self.heartbeat_task:
+            self.heartbeat_task = asyncio.create_task(self._heartbeat_loop())
+        
+        return client_id
+    
+    async def disconnect(self, client_id: str):
+        """Remove a WebSocket connection"""
+        if client_id in self.connections:
+            connection = self.connections.pop(client_id)
```

Ruff (0.12.2)

Local variable `connection` is assigned to but never used

Remove assignment to unused variable `connection`

(F841)